tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -20,90 +20,90 @@ public function testAnnotations()
         $class = $this->getReflectionClass();
         $reader = $this->getReader();
 
-        $this->assertCount(1, $reader->getClassAnnotations($class));
-        $this->assertInstanceOf($annotName = DummyAnnotation::class, $annot = $reader->getClassAnnotation($class, $annotName));
-        $this->assertEquals('hello', $annot->dummyValue);
+        self::assertCount(1, $reader->getClassAnnotations($class));
```

Comment:
Wait I was talking about the change from $this->assertCount to $self::assertCount not the change from assertTrue( , count()) to assertCount(). Or it's the same answer ?